frontend/packages/noobaa-storage-plugin/integration-tests/tests/e2e-tests/objectPages.scenario.ts

```diff
@@ -0,0 +1,46 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { execSync } from 'child_process';
+import { goToOBCPage, CreateOBCHandler, goToOBPage } from '../../views/obcPage.view';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { backingStore, bucketClass, storageClass, objectBucket } from '../../mocks/objectData';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  deleteResource,
+  untilNoLoadersPresent,
+} from '@console/internal-integration-tests/views/crud.view';
+import { OB_RESOURCE_PATH } from '../../utils/consts';
+
+describe('Test Object Bucket Claim CRUD', () => {
+  beforeAll(async () => {
+    execSync(`echo '${JSON.stringify(backingStore)}' | oc apply -f -`);
+    execSync(`echo '${JSON.stringify(bucketClass)}' | oc apply -f -`);
+    execSync(`echo '${JSON.stringify(storageClass)}' | oc apply -f -`);
+  });
+
+  afterAll(async () => {
+    execSync(`echo '${JSON.stringify(backingStore)}' | oc delete -f -`);
+    execSync(`echo '${JSON.stringify(bucketClass)}' | oc delete -f -`);
+    execSync(`echo '${JSON.stringify(storageClass)}' | oc delete -f -`);
```

Comment:
It's created as a part of ocs installation and should be destroyed accordingly.

Reply:
Same as above.